### DISCUSSION TITLE
为什么配置 kserve 的 资源拓扑config map ，没有效果

### DISCUSSION THREAD

**Discussion by sharewind** [kubevela/kubevela#6013] [Q&A]
> apiVersion: v1
kind: ConfigMap
metadata:
  name: kserve-relation
  namespace: vela-system
  labels:
    "rules.oam.dev/resource-format": "yaml"
    "rules.oam.dev/resources": "true"
data:
  rules: |-
    - parentResourceType:
        group: serving.kserve.io
        kind: InferenceService
      childrenResourceType:
        - apiVersion: serving.knative.dev/v1
          kind: Service
    - parentResourceType:
        group: serving.kserve.io
        kind: Service
      childrenResourceType:
        - apiVersion: serving.knative.dev/v1
          kind: Route
        - apiVersion: serving.knative.dev/v1
          kind: Configuration
    - parentResourceType:
        group: serving.kserve.io
        kind: Route
      childrenResourceType:
      - apiVersion: v1
        kind: Service
    - parentResourceType:
        group: serving.kserve.io
        kind: Configuration
      childrenResourceType:
      - apiVersion: serving.knative.dev/v1
        kind: Revision
    - parentResourceType:
        group: serving.kserve.io
        kind: Revision
      childrenResourceType:
        - apiVersion: apps/v1
          kind: Deployment

**Comment by sharewind** [comment]
W0520 14:44:08.209138    1868 tree.go:938] listing application resource tree has reached the max-depth 6 parentObject is {local apps/v1 ReplicaSet algo-dev model-example-predictor-default-00001-deployment-66bc65f4b8

**Reply by Somefive** [reply]
Ref #604
